app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -140,6 +140,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
 
         lifecycle.addObserver(webExtensionPopupFeature)
         StartupTimeline.onActivityCreateEndHome(this)
+
+        if(shouldStartInRecentsScreen(intent)) {
```

Comment:
You should run ./gradlew ktlint in your terminal to detect warnings and fix them, otherwise the PR checks won't pass. For example, you should add a space between `if` and `(`